modules/update-availability-faq.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+[id="channel-differences_{context}"]
+*What are the differences between each of the update channels?*
+
+* A new release is initially added to the `candidate` channel and is often available as an update target for clusters on the `candidate` channel.
+
+* After successful internal and external testing, a release on the `candidate` channel is supported by being promoted to the `fast` channel, and is often available to clusters on the `fast` channel.
+
+* After a delay and more successful testing, a release on the `fast` channel is finally promoted to the `stable` channel and is often available to clusters on the `stable` channel. This delay represents the only difference between the `fast` and `stable` channels.
++
+[NOTE]
+====
+For the latest z-stream releases, this delay may generally be a week or two. However, the delay for initial updates to the latest minor version may take much longer, generally 45-90 days.
+====
+
+* Releases promoted to the `stable` channel are simultaneously promoted to the `eus` channel.
+The primary purpose of the `eus` channel is to serve as a convenience for clusters performing an EUS-to-EUS update.
+
+[id="channel-safety_{context}"]
+*Is a release on the `stable` channel safer or more supported than a release on the `fast` channel?*
+
+* If a regression is identified for a release on a `fast` channel, it will be resolved and managed to the same extent as if that regression was identified for a release on the `stable` channel.
+
+* The only difference between releases on the `fast` and `stable` channels is that a release only appears on the `stable` channel after it has been on the `fast` channel for some time and has not had any major issues.
```

Comment:
Can we drop "and has not had any major issues" or replace it with text about giving update risks more time to bubble out?  We will add a release to stable channels after it has cooked in fast channels, even if we discover  major issues.  The only thing that is different is the delay, so customers in stable channels are more likely to have any conditional update risks already declared by the time it shows up in their channel.  For example:

1. Release A is all stable.
2. Release B enters candidate channels.
3. Risk X for A-to-B updates is discovered and declared as a conditional update risk.
4. Release B enters fast channels.
5. Risk Y for A-to-B updates is discovered and declared as a conditional update risk.
6. Release B enters stable channels.
7. Risk Z for A-to-B updates is discovered and declared as a conditional update risk.

Customers on release A in candidate channels may be recommended to update to B before any of those risks are known, and even before we know if B will ever be supported (it could end up getting tombstoned).

Customers on release A in fast channels may be recommended to update to B before risks Y and Z are known.

Customers on release A in stable channels may be recommended to update to B before risk Z is known.

The longer you wait, the more complete our understanding of A-to-B risks is.  The fast-to-stable wait is internal Red Hat for "we've collected a bunch of data now,  and are feeling pretty confident that any not-yet-discovered risks are very rare".

Reply:
Will replace "and has not had any major issues" with "which provides more time for new update risks to be discovered". Let me know if that doesn't work for you